src/components/HotspotEditorModal/index.js

```diff
@@ -0,0 +1,3 @@
+// We want to export the sub components as part of a single HotspotEditorModal module
+// Thay is why they are all re-exported from the index.js
+export { default as HotspotTooltipTab } from './HotspotTooltipTab/HotspotTooltipTab';
```

Comment:
On second thought, I think we should keep the API surface flat. `HotspotTooltipTab` is only to be used within `HotspotEditorModal` - we can achieve the same affordance a namespace provides instead through naming convention.

Since this is all in relation to editing hotspots, I think this component and others like it can bear the root "HotspotEditor", and suffix with a more specific name relating to form/function. So, I propose we rename `HotspotTooltipTab.jsx` -> `HotspotEditorTooltipTab.jsx` and delete this in favor of exporting as we normally do with all other components.

With this, the imports are well organized alphabetically too

```js
import {
  HotspotEditorModal
  HotspotEditorSomethingElse
  HotspotEditorTooltipTab
  ... etc
} from 'carbon-addons-iot-react';
```

Thoughts?

Reply:
I don't have much of an opinion here, I think either way is fine. I'll change it back :-)